CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ if (NOT Boost_ADDITIONAL_VERSIONS)
 endif()
 
 # first try to find the version
-find_package(Boost 1.50 REQUIRED COMPONENTS system filesystem thread ${BOOST_EXTRA})
+find_package(Boost 1.50 REQUIRED COMPONENTS system filesystem ${BOOST_EXTRA})
```

Comment:
If we don't need boost thread then `libboost-thread-dev` should be dropped Ubuntu/Debian and `libboost-thread1.55-dev` from travis.yml